lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -2292,6 +2292,41 @@ definitions:
           - 'NuRAN Cavium OC-LTE'
           - 'FreedomFi One'
         x-nullable: false
+      x2_enable_disable:
+        type: boolean
+        example: false
+        default: false
+      power_control:
+        $ref: '#/definitions/power_control'
+
+  power_control:
+    description: Config power control for enodeB.
+    type: object
+    properties:
+      reference_signal_power:
+        description: specified in dBm, with a numeric range between -60 and 50 inclusive.
+        type: integer
+        format: int32
+        minimum: -60
+        maximum: 50
+        example: 30
+      pa:
```

Comment:
Add some description, limits, and reference to spec if possible. Also for `pb` and `power_class`